roles/attribute-aggregation/tasks/main.yml

```diff
@@ -21,15 +21,21 @@
     - apachelink.conf
   notify: restart attribute-aggregationserver
 
+- name: Add the MariaDB docker network to the list of networks when MariaDB runs in Docker
+  ansible.builtin.set_fact:
+    aa_docker_networks:
+      - name: loadbalancer
+      - name: openconext_mariadb
+  when: mariadb_in_docker | default(false) | bool
+
 - name: Create and start the server container
   community.docker.docker_container:
     name: aaserver
     image: ghcr.io/openconext/openconext-attribute-aggregation/aa-server:{{ attribute_aggregation_server_version }}
     pull: true
     restart_policy: "always"
     state: started
-    networks:
-      - name: "loadbalancer"
+    networks: "{{ aa_docker_networks }}"
```

Comment:
Je vervangt hierdoor de list door een hash.  Moet de default niet zijn:
```yml
aa_docker_networks:
  - name: loadbalancer
```

Reply:
Klopt. Het aardige is dat het wel gewoon werkt, alles op TEST is met deze branch gedeployed :)